vowpalwabbitslim/Makefile

```diff
@@ -0,0 +1,50 @@
+
+vwslim_SOURCES = $(shell grep libvwslim_la_SOURCES Makefile.am | cut -d '=' -f 2)
+vwslim_OBJECTS = $(patsubst %.cc,%.o,$(vwslim_SOURCES))
```

Comment:
When I manually cd to vowpalwabbitslim and run "make things"  it fails with cannot find explore.h

Reply:
I don't think this is valid invocation. this depends on the flags set from the parent makefile

---
In reply to: [188391939](https://github.com/JohnLangford/vowpal_wabbit/pull/1481#discussion_r188391939) [](ancestors = 188391939)